### ISSUE TITLE
Canvas elements aren't aligned with other inline-block elements

### ISSUE THREAD

**Issue by jdm** [servo/servo#10724]
Firefox:
<img width="146" alt="screen shot 2016-04-19 at 6 19 31 pm" src="https://cloud.githubusercontent.com/assets/27658/14657343/6461b43c-065b-11e6-9c42-b7ffd6b3c61b.png">
Servo:
<img width="148" alt="screen shot 2016-04-19 at 6 20 07 pm" src="https://cloud.githubusercontent.com/assets/27658/14657351/6bff05a0-065b-11e6-99b7-d88ffeaada7b.png">

``` html
<style>
span {
  display: inline-block;
  width: 40px;
  height: 40px;
  border: 1px solid black;
}

canvas {
  border: 1px solid red;
}
</style>
<span></span>
<span></span>
<span></span>
<br>
<span></span>
<canvas width="40" height="40"></canvas>
<span></span>
<br>
<span></span>
<span></span>
<span></span>
```

**Comment by stshine** [comment]
This is fixed by #14490.

**Comment by jdm** [comment]
Thanks!
